c/validation/adbc_validation_util.cc

```diff
@@ -242,14 +242,18 @@ void CompareSchema(
   }
 }
 
-std::string GetDriverVendorVersion(struct AdbcConnection* connection,
-                                   struct AdbcError* error) {
+std::string GetDriverVendorVersion(struct AdbcConnection* connection) {
   const uint32_t info_code = ADBC_INFO_VENDOR_VERSION;
   const uint32_t info[] = {info_code};
 
   adbc_validation::StreamReader reader;
-  AdbcConnectionGetInfo(connection, info, 1, &reader.stream.value, error),
+  struct AdbcError error = ADBC_ERROR_INIT;
+  AdbcConnectionGetInfo(connection, info, 1, &reader.stream.value, &error),
       reader.GetSchema();
+  if (error.private_data != nullptr) {
+    throw std::runtime_error("error occured calling AdbcConnectionGetInfo!");
```

Comment:
```suggestion
    if (error.release) error.release(&error);
    throw std::runtime_error("error occured calling AdbcConnectionGetInfo!");
```
or just use a nullptr `error` (which should always be allowed)